sdks/java/io/common/pom.xml

```diff
@@ -33,10 +33,12 @@
       <dependency>
         <groupId>org.apache.beam</groupId>
         <artifactId>beam-sdks-java-core</artifactId>
+        <scope>test</scope>
```

Comment:
Hmm. correct, but odd that this is correct. I'd expect `HashingFn` to be in `main`, not `test` in that module. Irrelevant to this PR, but CC: @ssisk